app/models/elasticsearch_service.rb

```diff
@@ -69,7 +70,7 @@ def search_facilities(params)
     validate_search(params)
 
     size = params[:size].to_i
-    size = 50 if size == 0
+    size = 1000 if size == 0
```

Comment:
Yes, it makes sense. When testing this I just noticed that ElasticSearch seems to struggle with the amount of queries and the size of the response. We should probably think about debouncing these API calls. I opened #36 to track this problem.